src/ansys/fluent/core/streaming_services/events_streaming.py

```diff
@@ -545,6 +545,51 @@ def register_callback(
                 }
             return callback_id
 
+    def register_callback(
+        self,
+        event_types: (
+            SolverEvent
+            | MeshingEvent
+            | tuple[SolverEvent, ...]
```

Comment:
Allow both list and tuple, use Sequence as typehint

Reply:
Done. Thank you.